docs/pages/guide/01-start.md

```diff
@@ -1,6 +1,122 @@
+costly to build applications. When building applications, it takes time
+to design a system, wire together libraries, test your logic, handle
+security concerns, develop a great user interface, build features, and
+battle test your system. With Carbon, we aim to package fully featured
+and reusable experiences called Duplos®, or large Legos®, that can be
+installed in your application with a few commands.  The following
+diagram illustrates the Duplos available in Yavin today:</span>
+
+<span class="c0"></span>
+
+<img src="assets/images/Duplos_img.png" width="800" border=5px/>
+
+<span class="c18">Yavin Duplos</span>
+
+<span class="c0"></span>
+
+## <span class="c5">An Overview of Yavin’s Main Components</span>
+
+### <span class="c2">Denali Overview</span>
+
+### <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 145.47px; height: 39.28px;">![](assets/images/Denali_logo.png)</span>
+
+Yavin uses Denali's themeable design system. Denali was developed as a
+way to quickly create unified product families with intuitive user
+experiences. Denali components are built to be themeable by nature,
+which means we aren’t tied to their components visual design. For more
+information on denali, visit <span
+class="c17"><a href="https://www.google.com/url?q=http://denali.design&amp;sa=D&amp;ust=1606182367633000&amp;usg=AOvVaw3vxbzIhWRHGVWve5Y_S6bE" class="c3">denali.design</a></span><span
+class="c0">.</span>
+
+### <span class="c2">Elide Overview</span>
+
+### <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 2.67px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 131.51px; height: 43.84px;">![](assets/images/Elide_logo.png)</span>
+
+<span class="c0">Yavin uses Elide as its web service and data model
+building language. Elide is a Java library that lets you setup model
+driven GraphQL or JSON-API web services with minimal effort. Elide
+supports two variants of APIs:</span>
+
+1.  <span class="c0">A CRUD (Create, Read, Update, Delete) API for
+    reading and manipulating models.</span>
+2.  <span class="c0">An analytic API for aggregating measures over zero
+    or more model dimensions.</span>
+
+<span class="c0"></span>
+
+For more information on Elide, visit <span
+class="c17"><a href="https://www.google.com/url?q=http://elide.io&amp;sa=D&amp;ust=1606182367634000&amp;usg=AOvVaw2jtjGW202zysi6LjUbgJng" class="c3">elide.io</a></span><span
+class="c0">.</span>
+
+### <span class="c2">Navi Overview </span>
+
+<span
+style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 154.00px; height: 39.00px;">![](assets/images/Navi_logo.png)</span>
```

Comment:
I don't think we want to embed styling like this in the documentation.  Let's stick to markdown and let the style sheets render them.